atomic_reactor/plugins/pre_koji.py

```diff
@@ -36,15 +38,33 @@ def __init__(self, tasker, workflow, target, hub, root, proxy=None):
         self.pathinfo = koji.PathInfo(topdir=root)
         self.proxy = proxy
 
+    def _check_target_and_tag(self):
```

Comment:
Right, in fact we already have a check for that - https://github.com/release-engineering/koji-containerbuild/blob/develop/koji_containerbuild/plugins/builder_containerbuild.py#L529. 

Closing this as its now handled in k-cb